app/controllers/points_controller.rb

```diff
@@ -16,35 +16,21 @@ def index
 
   def new
     @point = Point.new
-    @services = Service.all
     @topics = Topic.all
     @cases = Case.all
-    if @query = params[:service_id]
```

Comment:
Is it necessary to remove this? Without it, it's not possible to preselect the service to create a point for (i.e. the "Add a point" button on a single service's page no longer works).